pkg/skaffold/deploy/resource/deployment.go

```diff
@@ -170,17 +171,17 @@ func (d *Deployment) cleanupStatus(msg string) string {
 	return clean
 }
 
-func parseKubectlRolloutError(err error) error {
+func parseKubectlRolloutError(err error) (proto.StatusCode, error) {
```

Comment:
It would be handy to have sample output for these conditions here.  Mostly in case they change and we're no longer matching the expected conditions.

Reply:
sounds good.